mathics/core/expression.py

```diff
@@ -748,6 +748,28 @@ def get_option_values(
                 option_values[name] = option.elements[1]
         return option_values
 
+    def get_rules_list(self) -> list:
```

Comment:
Since this happens just for Lists, which are always converted to `ListExpression`, why do not put this in that subclass?

Reply:
I tried that and I got an error in test_assignment.py. I think for expressions we need to do something. 

And was also not 100% certain we never use Expression when we mean ListExpression.